Content.Server/Anomaly/Components/TempReactingAnomalyComponent.cs

```diff
@@ -0,0 +1,63 @@
+using Robust.Shared.Utility;
+
+namespace Content.Server.Anomaly.Components;
+
+/// <summary>
+/// Causes the anomaly to react to different temperatures.
+/// </summary>
+/// <remarks>
+/// This could later be extended to react only to specific gasses.
+/// </remarks>
+[RegisterComponent]
+public sealed class TempReactingAnomalyComponent : Component
+{
+    /// <summary>
+    /// A list of reactions that happen at certain temperatures.
+    /// </summary>
+    [DataField("reactions")]
+    public List<TempReaction> Reactions = new();
```

Comment:
Should this just be an includedatadef instead.

Reply:
Assuming you mean `IncludeDataField` this does not seem to work with `List`. I just changed it to be `required: true`.